Pear/View/OnboardingSearchDropdownView.swift

```diff
@@ -124,6 +140,42 @@ class OnboardingSearchDropdownView: UIView {
         delegate?.updateSelectedFields(tag: tag, isSelected: true, valueSelected: title)
     }
 
+    func searchPlaces(place: String) {
+
+        /**
+         * Create a new session token. Be sure to use the same token for calling
+         * findAutocompletePredictions, as well as the subsequent place details request.
+         * This ensures that the user's query and selection are billed as a single session.
+         */
+
+        let token = GMSAutocompleteSessionToken.init()
+
+        // Create a type filter.
+        let filter = GMSAutocompleteFilter()
+        filter.type = .city
+
+        placesClient?.findAutocompletePredictions(
```

Comment:
Since you only use `placesClient` here there's no point in cluttering the class namespace. Just inline `GMSPlacesClient.shared()`.